app/views/shared/_errors.html.erb

```diff
@@ -0,0 +1,12 @@
+<% if @user.errors.any? %>
```

Comment:
This would be a problem when `@user` does not exist.

Maybe you can use ruby's `&.` operator as you previously did.

Hint 💡: You may have to use it twice